index.html

```diff
@@ -46,4 +45,241 @@ <h1 class="mola">  I'm Sayed Ramin.<br />Glad to see<br />
+                <button class="html" type="button">HTML</button>
+              </li>
+              <li class="btn">
+                <button class="css" type="button">CSS</button>
+              </li>
+              <li class="btn">
+                <button class="java" type="button">JAVASCRIPT</button>
+              </li>
+            </ul>
+            <div class="btn2">
+              <button type="button" class="btn-see">See Project</button>
+            </div>
+          </div>
+        </div>
+ <!--Works Section page 5-->
+ <section class="page5">
+  <div class="@continer">
+      </a>
+      <div class="pra1">
+        <h3 class="titleA">About My self</h3>
+        <p class="text3">
+          Hello I’m a software developer! I can<br> help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <hx class="call1">LET'S CONNECT</hx>
+        <ul class="socials1">
+            <li>
+                <a href="https://twitter.com/SayedRamin78773"><img src="image/icons/cat.svg" alt="github" /> </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/sayed-ramin-sadat-7770a0270/"><img
+                        src="image/icons/linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li>
+                <a href="#"><img src="image/icons/fingers.svg" alt="fingers" /></a>
+            </li>
+            <li>
+                <a href="https://github.com/sadat-Ramin"><img src="image/icons/twitter.svg" alt="twitter" /></a>
+            </li>
+            <li>
+                <a href="#"><img src="image/icons/m.svg" alt="M"/></a>
+            </li>
+        </ul>
+    </div>
+            <div class="btn3">
+              <button type="button" class="btn-get">Get my resume</button>
+                <ul class="accordion">
+                  <li>
+                    <div class="lan">
+                      <a class="title-link" href="#"
+                        ><span class="Language">Languages</span><div class="laa"><i class="fa-solid fa-chevron-down solid solids"></i></div></a
+                      >
+                    </div>
+                    <ul>
+                      <li class="skill">
+                        <img class="skillـimage" src="image/works/Ellipse 1.png" alt="" />
+                        <span class="skillـname">Javascript</span>
+                      </li>
+                      <li class="skill">
+                        <img class="skillـimage" src="image/works/Ellipse 1 (1).png" alt="" />
+                        <span class="skillـname">HTML</span>
+                      </li>
+                      <li class="skill">
+                        <img class="skill_image" src="image/works/Ellipse 1 (2).png" alt="" />
+                        <span class="skillـname">CSS</span>
+                      </li>
+                    </ul>
```

Comment:
Kindly reduce the space between these item as shown in the picture below 👍 , to stick to the Figma design

Your design
![Capture](https://github.com/sadat-Ramin/My_first_portfolio/assets/80458199/d8e84542-ce43-4ff6-8ac7-a7da06bb4782)

Figma design
<img width="156" alt="251546895-6c2c3509-600f-4154-a3b2-962c1540c5de" src="https://github.com/sadat-Ramin/My_first_portfolio/assets/80458199/bd950b1d-62b3-4ede-9a4a-d985795f2dbd">